taichi/backends/cc/codegen_cc.cpp

```diff
@@ -273,10 +275,18 @@ class CCTransformer : public IRVisitor {
   }
 
   void generate_serial_kernel(OffloadedStmt *stmt) {
-    emit_header("void Ti_{}(void) {{", kernel->name);
+    stmt->body->accept(this);
+  }
+
+  void generate_range_for_kernel(OffloadedStmt *stmt) {
+    // TI_ASSERT(stmt->const_begin && stmt->const_end);
+    auto begin_value = stmt->begin_value;
+    auto end_value = stmt->end_value;
+    auto var = define_var("int", stmt->raw_name());
+    emit("for ({} = {}; {} < {}; {} += {}) {{", var, begin_value,
+         stmt->raw_name(), end_value, stmt->raw_name(), 1 /* stmt->step? */);
```

Comment:
It seems to me that these are all the loop index?

```suggestion
         var, end_value, var, 1 /* stmt->step? */);
```

Reply:
No,
`var` is "int tmp233" when `C99_COMPAT = 0`.
`stmt->raw_name()` is "tmp233".